frontend/src/components/auth/Authy.tsx

```diff
@@ -40,40 +50,79 @@ const Authy = ({
     }
   };
 
+  const handlePasscode = (e: { target: { value: unknown } }) => {
+    const newPasscode = e.target.value as string;
+    if (newPasscode !== null && /^[0-9]{0,6}$/.test(newPasscode)) {
+      setPasscode(newPasscode);
+    }
+  };
+
+  const handleFocus = () => {
+    if (inputRef && inputRef.current) {
+      inputRef.current.focus();
+    }
+  };
+
   if (authenticatedUser) {
     return <Redirect to={HOME_PAGE} />;
   }
 
+  const boxIndexes = [0, 1, 2, 3, 4, 5];
+
   if (toggle) {
     return (
-      <div style={{ textAlign: "center" }}>
-        <h1>Authy</h1>
-        <form
-          onSubmit={(event) => {
-            event.preventDefault();
-            onAuthySubmit();
-          }}
-        >
-          <div>
-            <p>Enter token from Authy App</p>
-            <input
-              type="text"
-              value={passcode}
-              onChange={(event) => {
-                setPasscode(event.target.value);
-                setError("");
-              }}
-              placeholder="passcode"
-            />
-          </div>
-          <div>
-            <button className="btn btn-primary" type="submit">
-              Verify
-            </button>
-          </div>
-          {error && <p style={{ color: "red" }}>{error}</p>}
-        </form>
-      </div>
+      <>
+        <Box bg="teal.400" height="100vh" onClick={handleFocus}>
+          <Flex bg="white" height="100vh" width="47%" justifyContent="center" alignItems="center">
+            <VStack width="77%" align="flex-start" gap="2vh">
+              <Text variant="login">One last step!</Text>
+              <Text variant="loginSecondary">In order to protect your account, please confirm the authorization code sent to {email}</Text>
```

Comment:
Are we going to give them the last four digits of their number here? More of a design question, the Figma is kind of vague

Reply:
Correction: I think i'm not seeing an email because i didn't acc go through the entire sign up process, so it just shows up as blank right now ...

However, I think the email should still be censored, not sure if that is implemented or not.

<img width="520" alt="image" src="https://github.com/uwblueprint/supportive-housing/assets/75588919/c2c3b477-05af-4f94-ae4b-931fbfe96c0f">